app/helpers/external/mailbluster_helper.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'httparty'
+
+module External
+  module MailblusterHelper
+    MAILBLUSTER_URL = 'https://api.mailbluster.com/api/leads/'
+
+    # def query_api(user, method)
+    # TODO? General helper for all queries
+    # end
+
+    def create_lead(user)
+      options = {
+        headers: {
```

Comment:
I think the `headers` could be pulled out into its own default / const,  since it is the same for all requests.